### PR TITLE
docs(Main-modal): :memo: add doc's for main modal

### PR DESCRIPTION
**Pull request by VipinDevelops**
This pull request adds documentation for the main modal in the application.

 The added documentation outlines the usage of the main modal and the corresponding slash command ("/github") to open it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ The GitHub Rocket.Chat App provides a seamless integration between GitHub and Ro
 <h2 align='center'>💡 Usage 💡</h2>
 <ul>
     <li> See Helper Message / Command List -> /github help </li>
-    <li> The app can be accessed with any of the slash commands /gh or /github </li>
+    <li> The app can be accessed with any of the slash commands /gh or /github </li>
```
Comment (samad-yar-khan) [inline README.md:27]: @VipinDevelops These two should not be in bullet points.  Can you please fix these ?

Comment (VipinDevelops) [reply README.md]: I tied to impove the overall style of command list here is new one do let me know if any changes required 
image

**Comment by VipinDevelops** [comment]
[Screencast from 2024-02-23 19-53-06.webm](https://github.com/RocketChat/Apps.Github22/assets/99081689/35d29524-e78a-4647-9996-d190ccafe7fb)

**Comment by VipinDevelops** [comment]
Hey @samad-yar-khan  I update the Readme of the app according to new main modal also fixed few broken links

**Review by samad-yar-khan — COMMENTED** [review]
Thanks @VipinDevelops

**Review by samad-yar-khan — APPROVED** [review]

**Pull request merged**